src/ert/gui/model/snapshot.py

```diff
@@ -445,7 +464,7 @@ def _fm_step_data(
         if role == FileRole:
             data_name = FM_STEP_COLUMNS[index.column()]
             if data_name in {ids.STDOUT, ids.STDERR}:
-                return node.data.get(data_name, QVariant())
+                return node.data.get(data_name, None)
```

Comment:
Is `None` redundant here?

Reply:
ids.STDOUT and ids.STDERR are both optional in the typeddict. we need the None if they are not defined